### PR TITLE
Update from upstream

### PR DESCRIPTION
**Pull request by SilverFire**

### CODE REVIEW

File: jquery.pjax.js [hunk 1]
```diff
@@ -602,25 +600,15 @@ function cloneContents(container) {
...
-    .replace(/[?&]($|#)/, '\1')
-    .replace(/[?&]/, '?')
-}
-
+// Returns sanitized url.href String.
 function stripInternalParams(url) {
-  url = stripParam(url, '_pjax')
-  url = stripParam(url, '_')
-  return url
+  url.search = url.search.replace(/([?&])(_pjax|_)=[^&]*/g, '')
```
Comment (SilverFire) [inline jquery.pjax.js]: It seems that this fixes #15

File: .travis.yml [hunk 2]
```diff
@@ -8,3 +8,11 @@ script: script/test
 
 notifications:
   email: false
+deploy:
+  provider: npm
+  email: mislav.marohnic@gmail.com
+  api_key:
+    secure: nstPtsdxAHGPkKl2wOnMW4FXHg1nKxG/yCYQYqiC61JqhbfSaZFVjIQrdH+9GSel7F/K7VLwGZwSVgfiWUymWeEsNw8R/cyI9xCXf5iNF6WHIwuX6Q3Tnvb3Cx/bkeoc8+4v7qR172FtzwKWJq2iKXawd58/4zf2sduOwld1PMw=
```
Comment (samdark) [inline .travis.yml]: Hmm... what is it for?

Comment (SilverFire) [inline .travis.yml]: https://docs.travis-ci.com/user/deployment/npm/

Comment (samdark) [inline .travis.yml]: I guess that should not be in our fork...

Comment (SilverFire) [inline .travis.yml]: yes, we can drop it

**Comment by SilverFire** [comment]
Merged.

**Pull request merged**